openaerostruct/functionals/moment_coefficient.py

```diff
@@ -112,12 +115,15 @@ def compute(self, inputs, outputs):
             # normalize CM
             if j == 0:
```

Comment:
This isn't something you changed, but does this always assume that the main lifting surface (usually the wing) is the first one defined? If so, will the MAC, and thus the moment coefficient be wrong if the surfaces are defined in the wrong order? @kanekosh do you have any experience using multiple lifting surfaces? This seems error prone, but I'm not sure the best way to fix it...

Reply:
Yes. It assumes the first surface you define is the main wing. It's always been that way.

Reply:
Good catch... @sabakhshi how did you (or will you) deal with this for multisection surfaces? I think MAC of the 1st section doesn't make any sense if one physical surface is composed of multiple OAS "surfaces"?
Maybe we should move MAC computation into a separate component (and do some if-else there) and pass in the MAC as an input to this component?

Reply:
There is no need to worry about the multisection surfaces as they have been unified into a single surface before entering aeropoint.

Reply:
@sabakhshi Would you mind documenting this (moment coefficient is always normalized by the 1st surface's MAC), maybe in advanced_features/Multiple Lifting Surfaces? One sentence noting this down would be enough.